### PR TITLE
Remove requests workaround

### PR DESCRIPTION
**Pull request by dwighthubbard**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -23,12 +23,12 @@ version = 19.5.0
 [options]
 install_requires =
     jinja2
-    requests
+    requests>=2.22.0
```
Comment (sagarun) [inline setup.cfg]: Hopefully this still supports python 2.7

File: setup.cfg [hunk 2]
```diff
@@ -23,12 +23,12 @@ version = 19.5.0
...
     six>=1.5
     virtualenv
     wheel
+    urllib3
```
Comment (sagarun) [inline setup.cfg]: Same here. Does it support python 2.7?

File: setup.cfg (hunk 1 above)
Comment (dwighthubbard) [reply setup.cfg]: Yes it does

File: setup.cfg (hunk 2 above)
Comment (dwighthubbard) [reply setup.cfg]: Yes

**Pull request merged**